frontend/src/scenes/onboarding/Onboarding.tsx

```diff
@@ -4,141 +4,81 @@ import { useEffect } from 'react'
-                                        <Link
-                                            onClick={() => {
-                                                toggleIsPricingModalOpen()
-                                            }}
-                                        >
-                                            <span className="font-bold text-brand-red">Volume discounts</span>
-                                        </Link>{' '}
-                                        after {convertLargeNumberToWords(upgradePlan?.tiers?.[1].up_to, null)}/mo.
-                                    </p>
-                                )}
-                        </p>
-                        <ul>
-                            {pricingBenefits.map((benefit, i) => (
-                                <li className="flex mb-2 ml-6" key={`pricing-benefits-${i}`}>
-                                    <IconCheckCircleOutline className="text-success mr-2 mt-1" />
-                                    {benefit}
-                                </li>
-                            ))}
-                        </ul>
-                    </LemonCard>
-                    <LemonCard className="mt-8" hoverEffect={false}>
-                        <h2 className="text-3xl">Resources</h2>
-                        {product.docs_url && (
-                            <p>
-                                <Link to={product.docs_url}>
-                                    Documentation <IconOpenInNew />
-                                </Link>
-                            </p>
-                        )}
-                        <p>
-                            <Link to={communityUrl}>
-                                Community forum <IconOpenInNew />
-                            </Link>
-                        </p>
-                        <p>
-                            <Link to={tutorialsUrl}>
-                                Tutorials <IconOpenInNew />
-                            </Link>
-                        </p>
-                    </LemonCard>
-                </div>
-            </div>
-            <ProductPricingModal
-                modalOpen={isPricingModalOpen}
-                onClose={toggleIsPricingModalOpen}
-                product={product}
-                planKey={upgradePlan.plan_key}
-            />
-        </div>
+    return product ? (
+        <OnboardingWrapper>
+            <OnboardingProductIntro product={product} />
+            <SDKs usersAction="loading flags" />
+        </OnboardingWrapper>
+    ) : (
+        <></>
     )
 }
 
+const getOnboarding = (productKey: string): JSX.Element => {
+    if (productKey === 'product_analytics') {
+        return <ProductAnalyticsOnboarding />
+    } else if (productKey === 'session_replay') {
+        return <SessionReplayOnboarding />
+    } else if (productKey === 'feature_flags') {
+        return <FeatureFlagsOnboarding />
+    }
+    return <></>
```

Comment:
You'd have to add your product key here - probably some improvement here in terms of code quality but i think we'd need some sort of switch statement to get the correct flow